# Identification Of A Novel Glycolysis-related Gene Signature For Predicting Breast Cancer Survival
### **GSEA software usedd for the gene sets enrichment analysis of the MSigDB gene sets in breast cancer (Figure 1).**
### **R-code and r data used for analysis and visualization of breast cancer data.**
#### lasso for the constrution and calculation of signature for Figure 2A and 2B
#### Kaplanâ€“Meier survival analysis for Figure 3A, Figure 4Aa-4Da and Figure 5E-F, Figure 6(subgroup analysis)
#### Receiver operating characteristic curve analysis for Figure 3B,Figure 4Ab-4Db and Figure 5C
#### Nomogram and calibration plots for Figure 5B and 5D
#### Survival status, riskscore rank and heatmap for Figure 3C-E

### *Citation*
#### *Wang H, Lengerich BJ, Aragam B, Xing EP. Precision Lasso: accounting for correlations and linear dependencies in high-dimensional genomic data. Bioinformatics. 2019;35(7):1181-1187. doi:10.1093/bioinformatics/bty750*
#### *Subramanian A, Tamayo P, Mootha VK, et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A. 2005;102(43):15545-15550. doi:10.1073/pnas.0506580102*
#### *Lorent M, Giral M, Foucher Y. Net time-dependent ROC curves: a solution for evaluating the accuracy of a marker to predict disease-related mortality. Stat Med. 2014;33(14):2379-2389. doi:10.1002/sim.6079*
#### *Holleczek B, Brenner H. Model based period analysis of absolute and relative survival with R: data preparation, model fitting and derivation of survival estimates. Comput Methods Programs Biomed. 2013;110(2):192-202. doi:10.1016/j.cmpb.2012.10.004*

### ***The data that support the findings of this study are available from the corresponding author upon reasonable request.***
